src/test/resources/docker/create-infra.sh

```diff
@@ -5,6 +5,14 @@ set -ex
 db=$1
 
 case $db in
+
+  # percona xtradb cluster needs a bit more time to start
+  #"edb-9.5"|"edb-9.6"|"edb-10"|"edb-11"|"edb-12"|"edb-13")
```

Comment:
Will you clean this up and remove the commented code that's repeated on line 17?